### ISSUE TITLE
Documentation not descriptive enough

### ISSUE THREAD

**Issue by lennon101** [benjypng/logseq-tablerender-plugin#18]
Hey this looks like a great plugin, but I'm struggling to figure out how to render the table. I'm not sure if it's just me or not, but it feels like the documentation in the ReadMe could be a little more clearer. For example: you have written

>  Just use the slash command /Render table to get started!

So, I open a blank page and I try this and I get the following text: 
> {{renderer :tables_656915f0-5421-484e-bebf-87fea5bd1574}}

Now, what do I do? You have given an example but you haven't said where to put the  text? Should the /Render command come before everything else? Should I put the /Render as a separate block? Or do I insert the example as a child block to the /Render command? Lots of questions unanswered. 

You have written: 
>To draw a table above, you can use any one of the below approaches, with the flags data, rows or cols.

But where do I put these flags? as a child block to the text that is rendered with the /Render command? 

Then there is the basic stats. Again, where do I put the (`<shortcut>-<column>`)? Does it go as a child to all the other text? Or immediately below the /Render? inside the same block or on it's own. 

I think a full example or maybe a Gif would be great to see. 
Thanks!

**Comment by benjypng** [comment]
Would the snippets in the [Introduction](https://github.com/hkgnp/logseq-tablerender-plugin#basic-table) help?

**Comment by lennon101** [comment]
> Would the snippets in the [Introduction](https://github.com/hkgnp/logseq-tablerender-plugin#basic-table) help?

Not at all in my opinion. Because the snippets don't show where to put them? Are they supposed to be indented to the /render command? Or something else? Same goes for the examples given for the sum etc.

**Comment by benjypng** [comment]
Try the example below. 

The `{{renderer ...}}` is automatically generated when you type `/Render table`, so you just need to add the portion from `rows` onwards.

To add the mathematical formulas, add it to the same block as `rows`. In the example below, `sum-2` sums up the values in column 2 of the table.

```
- {{renderer(:tables_656a0198-9b78-4d81-ae9a-09811cce3c8d)}}
	- rows sum-2
		- hello
			- hello
		- **stuff**
			- *stuff*
		- [[John Tan]]
			- The quick brown fox jumps over the lazy dog
```

**Comment by lennon101** [comment]
Yeah thanks! I did end up figuring it all out. But I decided to raise this issue so that others wouldn't struggle with the same issues in the future. That information you've provided would be great to add in the documentation (IMO). 

i.e. 

This would be great information to include: 
---

The {{renderer ...}} is automatically generated when you type `/Render`, so you just need to add the portion from rows onwards.

To add the mathematical formulas, add it to the same block as rows. In the example below, sum-2 sums up the values in column 2 of the table.
```
- {{renderer(:tables_656a0198-9b78-4d81-ae9a-09811cce3c8d)}}
	- rows sum-2
		- hello
			- hello
		- **stuff**
			- *stuff*
		- [[John Tan]]
			- The quick brown fox jumps over the lazy dog
```
